### PR TITLE
Dockerize Application

### PR DESCRIPTION
**Pull request by clay-comer-wt**
Setup a Dockerfile and docker-compose.yml file for the application based on the example from [Turborepo](https://github.com/vercel/next.js/blob/canary/examples/with-docker/Dockerfile). 

**Docker Desktop**
Screenshot 2024-05-15 at 4 08 25 PM

**Local Browser**
Screenshot 2024-05-15 at 4 08 37 PM

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,6 +50,18 @@ To build all apps and packages, run the following command:
 npm run build
 ```
 
+### Running in Docker
+
+To run the application locally in a Docker container, run the following commands:
+
+```
+docker-compose build
```
Comment (laurenrevere) [inline README.md:63]: Should we mention somewhere in the readme that users need to install docker?

Or list it as a prereq?

Comment (clay-comer-wt) [reply README.md:63]: That's a good point. Its an implied prereq, however we should be explicit

Comment (clay-comer-wt) [reply README.md:63]: @laurenrevere updated!

Comment (nicholaslyon-dt) [reply README.md:63]: Should we also note about the zscaler issue? Otherwise looks good, got it to build now!

**Review by nicholaslyon-dt — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (clay-comer-wt) [reply README.md:63]: I'll add a quick note about it

Comment (clay-comer-wt) [reply README.md:63]: @nicholaslyon-dt updated!

**Pull request merged**